src/ontodia/diagram/view.ts

```diff
@@ -417,7 +417,7 @@ export class DiagramView extends Backbone.Model {
         }
     }
 
-    getLinkStyle(linkTypeId: string): LinkStyle {
+    public getLinkStyle(linkTypeId: string): LinkStyle {
```

Comment:
Don't add explicit `public` access modifier here (it's already `public` by default).